.github/CONTRIBUTING.MD

```diff
@@ -0,0 +1,24 @@
+# Contributing
+
+Thank for taking the time to checkout the repo and willing to contribute!
+
+Feature requests, bugs, and code improvements are welcome as contributions!
+
+If this is your first open source contribution, please have a look at [this](https://egghead.io/courses/how-to-contribute-to-an-open-source-project-on-github) guide.
+
+# Bug Reports & Feature Requests
+
+If you would like to submit a feature request or bug report, we encourage you to first look through the [issues](https://github.com/react-native-training/react-native-elements/issues) and [pull requests](https://github.com/react-native-training/react-native-elements/pulls) before filing a new issue.
+
+# Setup
+
+*  Install [Node.js](https://nodejs.org/) if you have not already.
+  *We advice you to use node 6.x.x*
+*  Fork the **react-native-elements** repo at (https://github.com/react-native-training/react-native-elements)[https://github.com/react-native-training/react-native-elements]
+* `git clone <your-forked-repo-url> && cd react-native-elements`
+* `yarn install` OR `npm install`
+* `npm test`
```

Comment:
Providing a link to https://github.com/react-native-training/react-native-hackathon-starter as a starter template would be useful